.github/workflows/build-and-publish.yaml

```diff
@@ -42,191 +42,50 @@ jobs:
-
-  qemu-builder-image:
-    if: ${{ contains(github.ref, 'redhat-') }}
-    name: Calculate Qemu Builder Image Digest
-    runs-on: 'ubuntu-latest'
-    outputs:
-      digest: ${{ steps.set-output.outputs.digest }}
-    env:
-      IMAGE_REGISTRY: quay.io/projectquay
-      TAG: main
-    steps:
-      - name: Pull Image
-        id: pull-image
-        run: docker pull "${IMAGE_REGISTRY}"/quay-builder-qemu:"${TAG}"
-      - name: Set Output
-        id: set-output
-        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/quay-builder-qemu:${TAG})"
-
-  operator-image:
-    if: ${{ contains(github.ref, 'redhat-') }}
-    name: Publish Operator Image
-    runs-on: 'ubuntu-latest'
-    outputs:
-      digest: ${{ steps.set-output.outputs.digest }}
-    env:
-      OPERATOR_IMAGE: quay.io/projectquay/quay-operator:${{needs.set-version.outputs.tag}}
-    needs: set-version
-    steps:
-      - name: Check out the repo
+      - name: check out the repo
         uses: actions/checkout@v2
         with:
           ref: ${{ github.event.inputs.branch || github.ref_name }}
 
-      - name: Login to Quay.io
-        uses: docker/login-action@v1
-        with:
-          registry: quay.io
-          username: ${{ secrets.QUAY_USER }}
-          password: ${{ secrets.QUAY_TOKEN }}
-
-      - name: Build and push
-        id: docker_build
-        uses: docker/build-push-action@v2
+      - name: install opm from OpenShift Mirror
+        uses: redhat-actions/openshift-tools-installer@v1
         with:
-          push: true
-          tags: ${{ env.OPERATOR_IMAGE }}
+          opm: "latest"
 
-      - name: Set digest output
-        id: set-output
+      - name: install yq
+        env:
+          VERSION: v4.14.2
+          BINARY: yq_linux_amd64
         run: |
-          docker pull "${OPERATOR_IMAGE}"
-          echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${OPERATOR_IMAGE})"
-
-  operator-index-images:
-    if: ${{ contains(github.ref, 'redhat-') }}
-    name: Publish Catalog Index Image
-    runs-on: 'ubuntu-latest'
-    needs: [quay-image, clair-image, builder-image, qemu-builder-image, operator-image, set-version]
-    env:
-      OPERATOR_NAME: quay-operator-test
-      BUNDLE: quay.io/projectquay/quay-operator-bundle
-      INDEX: quay.io/projectquay/quay-operator-index
-      TAG: ${{needs.set-version.outputs.tag}}
-    steps:
-      - name: Check out the repo
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ github.event.inputs.branch || github.ref_name }}
+          wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/local/bin/yq
+          chmod +x /usr/local/bin/yq
 
-      - name: Login to Quay.io
+      - name: login to Quay.io
         uses: docker/login-action@v1
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USER }}
```

Comment:
`REGISTRY` implied that it can be configured, but these variables imply that the registry have to be `quay.io`.

Reply:
true. I think we should have `REGISTRY` and `NAMESPACE`, so that both are configurable.
wdyt?

Reply:
done - went with my own suggestion